ha/cli/exec/nodeExecutor.py

```diff
@@ -43,14 +43,17 @@ def parse_cluster_args(self) -> bool:
         Parses the command line args.
         Return: argparse
         """
-        parser = argparse.ArgumentParser(prog='node start <Node> [all|server] [--json]')
+        parser = argparse.ArgumentParser(prog='node start',
+            usage = CLISchema.get_usage("node", "start"),
+            formatter_class = argparse.RawDescriptionHelpFormatter)
         parser.add_argument("node", help="Module")
         parser.add_argument("start", help="action to be performed")
-        parser.add_argument('--nodeid', help='Node to start', required=True)
-        group = parser.add_mutually_exclusive_group(required='True')
+        parser.add_argument("nodeid",
+            help="Provide node name to add it in cluster.", action="store")
```

Comment:
Please correct this help, the command here is for node start.

Reply:
Done